lib/nastavitve.dart

```diff
@@ -269,7 +269,7 @@ class NastavitvePageState extends State<NastavitvePage> {
                       backgroundColor: HexColor.fromHex("#8DD7F7"),
                     ),
                     title: 'O aplikaciji',
-                    subtitle: "Izvedi več o aplikaciji ŠCVApp",
+                    subtitle: "Podatki o aplikaciji ŠCVApp",
```

Comment:
"Izvedi več o aplikaciji" je dobeseden prevod "Find out more about the application". Tole moje je predlog za izboljšavo, če imata kakšno boljšo idejo, napišita.